content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+---
+slug: babette
+title: Call BEAST2 from R
+package_version: 2.0
+authors:
+  - Richèl J.C. Bilderbeek
+date: 2019-01-06
+categories: blog
+topicid: 
+tags:
+  - Software Peer Review
+  - R
+  - community
+  - software
+  - packages
+  - phylogeny
+  - phylogenetics
+  - BEAST
+  - BEAST2
+  - Bayesian
+---
+
+`babette` [^1] is a package to work with BEAST2 [^2], 
+a software platform for Bayesian evolutionary analysis from R.
+
+`babette` is a spin-off of my own academic research.
+As a PhD I work on models of diversification: mathematical descriptions
+of how species form new species. Instead of working on a species'
+individuals, I work on species as evolutionary lineages.
+A good way to show the evolutionary relationships between species
+are phylogenies. For example, this a phylogeny, with four species,
+eloquently named '1', '2', '3' and '4':
+
+![A phylogeny with four species](img/blog-images/2019-01-06-babette/phylogeny.png)
```

Comment:
We now recommend using [Hugo shortcodes](https://gohugo.io/content-management/shortcodes/#use-hugos-built-in-shortcodes) to add images. 

Example to be modified: `{{< figure src = "/img/blog-images/2019-06-04-post-filename/name-of-image.png" width = "200" alt = "this is alternative text that describes the image" >}}`

We now recommend that every image be accompanied by alternative text (alt="some text") to make it more accessible to people with disabilities and provide a better user experience for everyone. The alternative text should convey the meaning or content that is displayed in the image. For information on what should go in alternative text, refer to [this tutorial](https://www.w3.org/WAI/tutorials/images/informative/).

Reply:
Updated all images :+1: